### ISSUE TITLE
On my Mac. I had canceled out...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#1088]
Notes
======
On my Mac. I had canceled out of starting a project and importing a project multiple times. I then tried to open my Psalms project. When the loading dialog went away, I saw a blank window for a minute or more. At that point, clicking on the 3-dot menu opened the menu, but selecting any menu option didn't do anything. Eventually the Psalms project appeared and now I can send feedback.

Environment
======
Environment Key | Value
:--: | :--:
Version |11.1.0
Operating System | Darwin
Platform | darwin
Release | 15.6.0
Architecture | x64

Log History
======
```javascript
2017-02-14 16:27:4 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-14 16:27:4 I/migrator.js:31:2: Migrated projects
	[]
2017-02-14 17:1:13 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-14 17:1:13 I/migrator.js:31:2: Migrated projects
	[]
2017-02-16 10:18:46 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-16 10:18:46 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1th_text_ulb",
	  "es-419_luk_text_ulb"
	]
2017-02-16 11:47:10 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-16 11:47:11 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1th_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-17 09:8:29 E/ts-dashboard.html:950:3: Upload Failed
	{
	  "status": 500,
	  "data": "{\"message\":\"updateUser: e-mail has been used [email: qa98@gmail.com]\",\"url\":\"https://godoc.org/github.com/go-gitea/go-sdk/gitea\"}"
	}
2017-02-20 17:44:54 E/ts-dashboard.html:847:3: Import Failed
	Error: Not a resource container
2017-02-20 17:48:46 E/ts-dashboard.html:847:3: Import Failed
	Error: Not a resource container
2017-02-20 17:53:42 E/ts-dashboard.html:847:3: Import Failed
	Error: Not a resource container
2017-02-21 09:44:52 E/ts-dashboard.html:847:3: Import Failed
	Error: Not a resource container
2017-02-21 09:58:26 E/ts-dashboard.html:847:3: Import Failed
	Error: Not a resource container
2017-02-21 10:1:47 E/ts-dashboard.html:847:3: Import Failed
	Error: Not a resource container
2017-02-21 10:6:49 E/ts-dashboard.html:869:3: Import Failed
	Error: Missing required property "translate_mode"
2017-02-21 10:13:51 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:13:51 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:13:52 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 10:26:10 E/ts-dashboard.html:869:3: Import Failed
	Error: Missing required property "checking_level"
2017-02-21 10:37:29 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:37:29 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:37:29 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 10:37:29 E/ts-dashboard.html:1193:2: Error updating projects
	Resource container does not exist
2017-02-21 10:39:4 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:39:4 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:39:5 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 10:39:5 E/ts-dashboard.html:1193:2: Error updating projects
	Resource container does not exist
2017-02-21 10:39:55 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:39:55 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:39:55 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_reg",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 10:39:55 E/ts-dashboard.html:1193:2: Error updating projects
	Resource container does not exist
2017-02-21 10:44:14 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:44:14 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:44:15 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 10:46:40 E/projects.js:260:26: 
	Error: ENOENT: no such file or directory, open '/Users/cckozie/Library/Application Support/translationstudio/library/resource_containers/new_phm_mine/content/toc.yml'
2017-02-21 10:50:48 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:50:48 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:50:48 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 10:54:15 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:54:15 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:54:16 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 10:58:10 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 10:58:10 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 10:58:10 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 11:26:31 I/ts-dashboard.html:837:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project en-x-demo1_1ch_text_ulb
	Error: Failed to migrate project
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project en-x-demo1_1th_text_ulb
	Error: Failed to migrate project
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project en-x-demo1_col_text_ulb
	Error: Failed to migrate project
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project en-x-demo1_exo_text_ulb
	Error: Failed to migrate project
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project en-x-demo1_psa_text_ulb
	Error: Failed to migrate project
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project es-419_luk_text_ulb
	Error: Failed to migrate project
2017-02-21 11:26:31 W/migrator.js:24:2: Unable to migrate project qaa-x-1560c6_tit_text_reg
	Error: Failed to migrate project
2017-02-21 11:26:32 W/migrator.js:24:2: Unable to migrate project qaa-x-731b78_phm_text_reg
	Error: Failed to migrate project
2017-02-21 11:26:32 I/migrator.js:31:2: Migrated projects
	[]
2017-02-21 11:39:9 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 11:39:9 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 11:39:10 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 11:40:18 E/projects.js:260:26: 
	TypeError: Cannot read property 'forEach' of undefined
2017-02-21 11:42:37 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 11:42:37 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 11:42:38 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 11:43:8 E/projects.js:260:26: 
	TypeError: Cannot read property 'forEach' of undefined
2017-02-21 11:44:56 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 11:44:56 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 11:44:57 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-21 11:44:58 E/projects.js:260:26: 
	TypeError: Cannot read property 'forEach' of undefined
2017-02-21 11:45:25 E/projects.js:260:26: 
	TypeError: Cannot read property 'forEach' of undefined
2017-02-21 11:54:27 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-21 11:54:27 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-21 11:54:27 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-22 12:0:25 E/ts-dashboard.html:655:3: Import Failed
	TypeError: Cannot read property 'indexOf' of undefined
2017-02-22 12:2:11 I/ts-dashboard.html:395:28: Chunk Conflicts
	[
	  "01-01",
	  "01-03",
	  "01-07",
	  "01-09",
	  "01-12",
	  "01-15",
	  "01-18",
	  "01-20",
	  "01-22",
	  "01-25",
	  "01-28",
	  "02-01",
	  "02-03",
	  "02-05",
	  "02-09",
	  "02-12",
	  "02-14",
	  "02-17",
	  "02-19",
	  "02-22",
	  "02-25",
	  "02-28",
	  "03-01",
	  "03-04",
	  "03-06",
	  "03-08",
	  "03-12",
	  "03-15",
	  "03-17",
	  "03-20",
	  "04-01",
	  "04-04",
	  "04-08",
	  "04-10",
	  "04-14",
	  "04-18",
	  "04-21"
	]
2017-02-22 12:6:32 I/ts-dashboard.html:395:28: Chunk Conflicts
	[
	  "01-01",
	  "01-03",
	  "01-07",
	  "01-09",
	  "01-12",
	  "01-15",
	  "01-18",
	  "01-20",
	  "01-22",
	  "01-25",
	  "01-28",
	  "02-01",
	  "02-03",
	  "02-05",
	  "02-09",
	  "02-12",
	  "02-14",
	  "02-17",
	  "02-19",
	  "02-22",
	  "02-25",
	  "02-28",
	  "03-01",
	  "03-04",
	  "03-06",
	  "03-08",
	  "03-12",
	  "03-15",
	  "03-17",
	  "03-20",
	  "04-01",
	  "04-04",
	  "04-08",
	  "04-10",
	  "04-14",
	  "04-18",
	  "04-21"
	]
2017-02-23 09:54:11 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 09:54:12 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 09:54:12 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 10:34:0 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 10:34:1 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 10:34:1 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 10:34:2 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 10:36:42 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 10:36:42 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 10:36:43 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 10:36:53 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 10:37:48 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 10:38:17 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 10:38:17 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 10:38:17 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 10:38:18 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 10:38:36 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 10:39:12 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 12:44:59 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 12:44:59 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 12:45:1 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 12:45:4 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 13:55:57 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 13:55:57 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 13:55:58 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 13:55:59 E/projects.js:260:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 13:56:32 E/projects.js:260:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 13:57:17 E/projects.js:260:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 13:58:55 E/projects.js:260:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 14:0:16 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 14:0:16 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 14:0:16 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 14:0:17 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 14:1:19 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 14:1:19 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 14:1:20 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 14:1:20 E/projects.js:260:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 14:55:27 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-23 14:55:27 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-23 14:55:28 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-23 14:55:28 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 14:57:40 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-23 14:59:40 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 10:18:28 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-24 10:18:28 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-24 10:18:30 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-24 10:18:32 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 10:49:6 I/ts-dashboard.html:1208:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-24 10:49:6 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-24 10:49:7 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-24 10:49:7 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 10:49:35 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 10:49:39 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 12:14:55 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 14:39:21 I/ts-dashboard.html:1278:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-24 14:39:21 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-24 14:39:22 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-24 14:39:23 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 14:42:46 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-24 15:19:40 E/ts-dashboard.html:655:3: Import Failed
	Error: ENOENT: no such file or directory, open '/Users/cckozie/Library/Application Support/translationstudio/temp/Runyankole/manifest.json'

```

**Comment by cckozie** [comment]
I closed and reopened tS, and now it opens the project normally.

**Comment by cckozie** [comment]
I think it may not have anything to do with all of the canceling. All of that was part of trying to find LTR projects. I imported one from door43 and noticed that scrolling through the project was very slow. I was thinking maybe it had to do with the Arabic font, but now I see that after restarting tS, that project scrolls normally. All of this testing was done right after installing build 114.

**Comment by cdwhitfield73** [comment]
Based on this issue, I made two changes.  Both of which will help, but I am not 100% sure it will solve everything.  I made another change to the timing of the loading dialog that should ensure it will stay active until the project is completely loaded, so if the blank window takes a minute or more before the project is displayed, the loading box will stay open that whole time.  While this fixes the issue of the user being able to do stuff during that time, I am not sure what caused Psalms to lock up for a minute on loading.  I am also not sure what caused the Arabic project to scroll slowly the first time.  However, these "could" be related to the error that is showing up in this log that I have never seen before "TypeError: Cannot read property 'status' of null".  I have fixed the root cause of this error, which will prevent it from occurring again and MAY eliminate the cause of the two slow downs.
